hello.apex

```diff
@@ -1,17 +1,17 @@
-# You will set your LaunchDarkly SDK key during the bridge.
+// You will set your LaunchDarkly SDK key during the bridge.
 
```

Comment:
While we're at it, can you change this to `You will set your LaunchDarkly SDK key through the bridge application.`